state/azure/cosmosdb/cosmosdb.go

```diff
@@ -50,6 +51,7 @@ type CosmosItem struct {
 	Value        interface{} `json:"value"`
 	IsBinary     bool        `json:"isBinary"`
 	PartitionKey string      `json:"partitionKey"`
+	TTL          *int        `json:"ttl,omitempty"`
```

Comment:
wonder why it is a pointer? If for performance, why `PartitionKey` not?

Reply:
This field is optional and there is a difference between 0 and nil TTL.

Reply:
I learn.